src/test/shared/sam/cli/samCliContext.test.ts

```diff
@@ -1,21 +0,0 @@
-/*!
- * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
- * SPDX-License-Identifier: Apache-2.0
- */
-
-import * as assert from 'assert'
-import { getSamCliDockerImageName } from '../../../../shared/sam/cli/samCliContext'
```

Comment:
looks like getSamCliDockerImageName was not removed from the codebase

Reply:
Opps, yep. I meant to remove it